packages/next/src/build/webpack/loaders/next-flight-loader/index.ts

```diff
@@ -95,7 +95,7 @@ export { e${cnt++} as ${ref} };`
   if (buildInfo.rsc?.type !== RSC_MODULE_TYPES.client) {
     if (noopHeadPath === this.resourcePath) {
       warnOnce(
-        `Warning: You're using \`next/head\` inside the \`app\` directory, please migrate to the Metadata API. See https://nextjs.org/docs/app/api-reference/file-conventions/metadata for more details.`
+        `Warning: You're using \`next/head\` inside the \`app\` directory, please migrate to the Metadata API. See https://nextjs.org/docs/app/building-your-application/optimizing/metadata for more details.`
```

Comment:
@delbaoliveira what do you think about using this link instead? https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration#step-3-migrating-nexthead

That tells the story of why `next/head` is no longer necessary.

Reply:
```suggestion
        `Warning: You're using \`next/head\` inside the \`app\` directory, please migrate to the Metadata API. See https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration#step-3-migrating-nexthead for more details.`
```

Reply:
Good suggestion!